api-server/routes/cats.js

```diff
@@ -0,0 +1,13 @@
+const router = require("express").Router()
+
+module.exports = db => {
```

Comment:
What benefits do you get from passing `db` to your route handler?

Reply:
By passing db to the route handlers, I was attempting to mock/simulate the structure that would have existed had I used a PostgreSQL connection pool object. If using a pool object, I would have instantiated my database connection which would then be passed as the db parameter to the route handlers. While it does not directly serve a functional purpose in the current state of the app, it would make it easier if upgrading to use an actual database in the future since the connection pool could be used directly within the route handlers to query the database.